tracing/tracing.go

```diff
@@ -48,22 +45,26 @@ var otHeaders = []string{
 	flags,
 }
 
+func valueFromCarrier(carrier opentracing.TextMapCarrier, key string) string {
```

Comment:
function comment please, specifically stating that when a key is not found on the map that the value returned is an empty string.